Sources/ProjectDescription/Template.swift

```diff
@@ -0,0 +1,76 @@
+public struct Template: Codable {
+    public let description: String
```

Comment:
Did you check if the templates are part of the project generated with the `tuist edit` command?

Reply:
At the moment it does not seem so - but it probably should if I am not mistaken? 🙂 If so, I'll add it to the implementation check list.